samples/cpp/benchmark_app/main.cpp

```diff
@@ -268,6 +268,17 @@ int main(int argc, char* argv[]) {
         }
 
         bool perf_counts = false;
+        // check if using the virtual device
+        auto if_auto = device_name.find("AUTO") != std::string::npos;
```

Comment:
What if my own plugin/device will be names MYAUTODEVICE? We should parse device name carefully taking into account semicolon and commas. Moreover this parsing is done already in `parse_devices`